specifyweb/stored_queries/format.py

```diff
@@ -340,6 +340,9 @@ def _fieldformat(self, specify_field: Field,
             return cast(field,
                         types.Numeric(65))  # 65 is the mysql max precision
 
+        if isinstance(field.comparator.type, types.JSON):
```

Comment:
won't the `specify_field.type` be "json" (so you won't have to add an additional sqlalchemy_type)?

Reply:
hmm... `isinstance(field.comparator.type, types.JSON)` this is working.
I see what you mean `field.comparator.type` -> `JSON()`

I could do
```
from sqlalchemy.sql.sqltypes import JSON
isinstance(field.comparator.type, JSON)
```
but I don't really see that as any better of a solution

Reply:
`specify_field`. that's the field defined in the datamodel (NOT django OR SQLAlchemy). It should be of type json. If not, make it.